### PR TITLE
Create simple motion capture relay node for the demo

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: nodes/motion_capture_relay.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+
+class MoCapRelay:
+    def __init__(self, name: str) -> None:
+        self._node_name = name
+        rospy.init_node(self._node_name, anonymous=False)
+
+        if not rospy.has_param("~sphere_radious"):
+            rospy.logerr(
+                f"[{rospy.get_name()}] Param ~sphere_radious not fount, but required. Shutting down!"
```
Comment (ahaffemayer) [inline nodes/motion_capture_relay.py]:
```suggestion
                f"[{rospy.get_name()}] Param ~sphere_radious not found, but required. Shutting down!"
```

File: README.md [hunk 2]
```diff
@@ -110,6 +110,18 @@ roslaunch panda_torque_mpc real_controllers.launch controller:=ctrl_mpc_lineariz
 ROS_NAMESPACE=/ctrl_mpc_linearized rosrun panda_torque_mpc crocoddyl_motion_server_node
 ```
 
+### Run with motion capture
+
+Select `motion_capture.launch` paramter `relay` to **false** to crate fake motion capture input data.
+
```
Comment (ahaffemayer) [inline README.md]:
```suggestion
Select `motion_capture.launch` parameter `relay` to **false** to create fake motion capture input data.

```

**Review by ahaffemayer — CHANGES_REQUESTED** [review]
good

**Comment by MedericFourmy** [comment]
Should this be closed after the coding week work?

**Comment by Kotochleb** [comment]
@MedericFourmy definitively this should be closed as it is redundant to the https://github.com/agimus-project/panda_torque_mpc/pull/10 which is also currently developed fork

**Pull request closed without merging**